index.js

```diff
@@ -89,4 +97,25 @@ module.exports = function (processors, options) {
   }
 
   return stream
+})
+
+
+function withConfigLoader(cb) {
+  return function (plugins, options) {
+    if (typeof plugins === 'undefined') {
+      return cb(postcssLoadConfig)
```

Comment:
I don't want to pass options to the config loader, because this goes against the point of having this config in the first place. Anything that is sent to `postcssLoadConfig` can become a local constant in this file.